README.md

```diff
@@ -425,6 +432,24 @@ Once you have stripped results, if you ran an experiment in reboot mode you
 will need to reboot manually to re-run the stripped results (with rc.local set
 up correctly).
 
+## Security Notes
+
+Krun is not designed to be run on a production machine, as it uses sudo to
```

Comment:
I'm not sure "not on a production machine" is right -- we do want it run on a nice clean machine! But we could maybe say "not on a machine intended for secure multi-user useage" or similar?